Source/Blazorise/Interfaces/Modules/IJSDropdownModule.cs

```diff
@@ -19,9 +19,10 @@ public interface IJSDropdownModule : IBaseJSModule,
         /// <param name="targetElementId">Target element for the dropdown menu.</param>
         /// <param name="altTargetElementId">An alternative target element for the dropdown menu, usually for split dropdowns.</param>
         /// <param name="menuElementId">Dropdown menu element id.</param>
+        /// <param name="showElementId">Id for the target element that will be listening to the 'show event'.</param>
         /// <param name="options">Additional options for the button initialization.</param>
         /// <returns>A task that represents the asynchronous operation.</returns>
-        ValueTask Initialize( ElementReference elementRef, string elementId, string targetElementId, string altTargetElementId, string menuElementId, object options );
+        ValueTask Initialize( ElementReference elementRef, string elementId, string targetElementId, string altTargetElementId, string menuElementId, string showElementId, object options );
```

Comment:
Seems to me that `showElementId` is the same as `menuElementId`.

Reply:
Hmm maybe not...

Reply:
Yes, Bulma has to override.

Reply:
> All is working now except for AntDesign. When you open menu first time it works, but as soon as you close and reopen it it show like:
> 
> ![image](https://user-images.githubusercontent.com/900302/162614784-8068febe-f8fd-44f9-83b8-a10d8079306f.png)

I think I broke it on merge or smthg, fixing it....